R/appshot.R

```diff
@@ -2,6 +2,8 @@
 #'
 #' @inheritParams webshot
 #' @param app A Shiny app object, or a string naming an app directory.
+#' @param file Name of output file. Should end with \code{.png}, \code{.pdf}, or
```

Comment:
I don't think this is necessary, because `appshot`'s documentation should inherit values from `webshot`.